app/views/shared/_footer.html.haml

```diff
@@ -107,10 +108,9 @@
         %p.text-small
           = t '.footer_legal_text_html', {content_license: link_to('CC BY-SA 3.0', 'https://creativecommons.org/licenses/by-sa/3.0/'), code_license: link_to('AGPL 3', 'https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)' )}
         %p.text-small
-          %div
```

Comment:
If you look at the failed test's output @drummer83 

> Failure/Error: find("div > a", text: "cookies policy").click
     
     Capybara::ElementNotFound:
       Unable to find css "div > a"
     # ./engines/web/spec/features/consumer/cookies_spec.rb:158:in `click_footer_cookies_policy_link_and_wait'
     # ./engines/web/spec/features/consumer/cookies_spec.rb:41:in `block (4 levels) in <top (required)>'
     # ./engines/web/spec/features/consumer/cookies_spec.rb:10:in `block (3 levels) in <top (required)>'

This is the root cause. We need to decouple the test from the markup so it doesn't break.

Reply:
Thank you, @sauloperez!
Sorry, I am newbie...